symbolic-minidump/benches/from_minidump.rs

```diff
@@ -0,0 +1,39 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use symbolic_common::ByteView;
+use symbolic_minidump::cfi::CfiCache;
+use symbolic_minidump::processor::{FrameInfoMap, ProcessState};
+use symbolic_testutils::fixture;
```

Comment:
Please group imports according to our coding guidelines:

```suggestion
use std::fs::File;
use std::io::{BufRead, BufReader};

use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};

use symbolic_common::ByteView;
use symbolic_minidump::cfi::CfiCache;
use symbolic_minidump::processor::{FrameInfoMap, ProcessState};
use symbolic_testutils::fixture;
```

Reply:
Sorry about that!